### PR TITLE
Datahub

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**
### Merge Notes
- Increment version

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -101,11 +102,17 @@ function AppContent({ Component, pageProps }: AppProps<AppCommonProps>) {
             </EvmProvider>
           </ErrorBoundary>
         </div>
+        <PostSubscriber />
       </QueryProvider>
     </ThemeProvider>
   )
 }
 
+function PostSubscriber() {
```
Comment (samchuk-vlad) [inline src/pages/_app.tsx:111]: Strange component. Can we just use this hook on line 74?

File: src/server/ipfs.ts [hunk 2]
```diff
@@ -30,9 +30,9 @@ export function getIpfsApi() {
   return {
     ipfs,
     saveAndPinJson: async (content: Record<any, any>) => {
-      // const cid = await ipfs.saveContentToOffchain(content as any)
-      const cid = await ipfs.saveJson(content)
-      await ipfs.pinContent(cid, props)
+      const cid = await ipfs.saveContentToOffchain(content as any)
+      // const cid = await ipfs.saveJson(content)
+      // await ipfs.pinContent(cid, props)
```
Comment (samchuk-vlad) [inline src/server/ipfs.ts:35]: Remove this lines

File: src/services/subsocial/datahub/mappers.ts [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    ownerId: post.ownedByAccount.id,
+    upvotesCount: 0,
+    contentId: post.content ?? '',
+    repliesCount: 0,
+    sharesCount: 0,
+    spaceId:
+      post.space?.persistentId ?? post.rootPost?.space?.persistentId ?? '',
+    isUpdated: !!post.updatedAtTime,
+    rootPostId: post.rootPost?.persistentId ?? '',
+    // TODO: update followers count
```
Comment (samchuk-vlad) [inline src/services/subsocial/datahub/mappers.ts:112]: Do you plan to do it in future? Or it should be done in this release?

**Review by samchuk-vlad — APPROVED** [review]

File: src/pages/_app.tsx (hunk 1 above)
Comment (teodorus-nathaniel) [reply src/pages/_app.tsx:111]: I have to add this because this hook needs to be inside of query client provider, so I can't just call it inside _app

**Pull request merged**